example/ipsec_crypto/odp_ipsec_stream.c

```diff
@@ -139,6 +139,27 @@ int create_stream_db_entry(char *input)
 	return 0;
 }
 
+static const void *get_evp_md(odp_auth_alg_t auth)
```

Comment:
I think this should return const EVP_MD *. Then there would be no need for any casting in this function or elsewhere.

Reply:
Ack. Fixed in v6.